src/edu/cs505/g5/RasberriPiState.java

```diff
@@ -0,0 +1,23 @@
+package MagicMirror2;
+
+/**
+ * This class shows different states of RasberriPi
+ * @author CS505-Group5 
+ */
+public class RasberryPiState {

```

Comment:
Compile error.  Class does not match file name RasberriPiState.java

Reply:
modified